aips/aip-116.md

```diff
@@ -120,22 +112,22 @@ The `domain` field may be omitted from the `AptosSignInInput` to prevent the cre
 
```

Comment:
A little bit confused by the comment above:

> "The `domain` field may be omitted from the `AptosSignInInput` to prevent the creation of application-specific phishing messages."

This field should **never** be omitted, no? And that's how you prevent phishing. (Not sure it makes sense to distinguish "phishing" as "application-specific"...)

Reply:
I still don't understand what you are trying to say here:
> The interface for the signing request sent to the wallet may have a variant of the `AptosSignInInput` interface that omits the `domain` field to prevent the creation of application-specific phishing messages.

Reply:
Oh, I think I understand but only cause I found this later on below:

> The domain field has been omitted from the AptosSignInInput interface to prevent the creation of application-specific phishing messages.

I am still rather confused though: if you don't have a domain field there anymore, then your verification logic must take the domain separately... which is a bit ugly. I will look below to understand.